MoviesApp/MoviesApp/SecondScreen/FilmInfoPresenter/FilmInfoPresenter.swift

```diff
@@ -2,29 +2,41 @@
 // Copyright © RoadMap. All rights reserved.
 
 import Foundation
+import RealmSwift
 
 /// Презентер FilmInfoTableVC
 final class FilmInfoPresenter: FilmsInfoTableViewPresentable {
     // MARK: - Private properties
 
-    private let networkService: any NetworkServicable
+    private let networkService: NetworkServicable
 
     private weak var view: FilmInfoViewProtocol?
+    private var realmService: RealmServicable
     private var router: Routable?
 
     // MARK: - Public properties
 
-    var actors: [Actor]?
+    var actors: Results<Actor>?
     var trailersResults: [TrailerInfo]?
+    var imageService: ImageServicable
```

Comment:
```suggestion
    var imageService: ImageServicable
```
Это все сделай приватным и передавай через инит в роутере